### PR TITLE
[#2125] Fixed failing Page getHTML tests

### PR DESCRIPTION
**Pull request by mjschranz**
https://webmademovies.lighthouseapp.com/projects/65733-popcorn-maker/tickets/2125-page-gethtml-tests-failing

### CODE REVIEW

File: test/page/page-getHTML.html [hunk 1]
```diff
@@ -31,8 +31,8 @@
               testDiv = document.createElement( "div" ),
               headString = html.substring( html.indexOf( "<head>" ) + 6, html.lastIndexOf( "</head>" ) ),
               expectedBaseSrc = location.href.substring( 0 , location.href.indexOf( "/page/" ) + 6 ),
-              bodyString = html.substring( html.lastIndexOf( "<body class=\"tray-minimized\">" ), html.lastIndexOf( "</body>" ) );
-
+              bodyString = html.substring( html.lastIndexOf( "<body class=\"\">" ), html.lastIndexOf( "</body>" ) );
+console.log(bodyString);
```
Comment (secretrobotron) [inline test/page/page-getHTML.html]: Please remove.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/01bfde67129694c/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @ScottDowne received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/b70affbe17c7cf4/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/01bfde67129694c/output.txt

Total script time: 2.00 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/b70affbe17c7cf4/output.txt

Total script time: 2.00 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @ScottDowne received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/cdde28799045a47/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @ScottDowne received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/b555d9d512f3ac9/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/cdde28799045a47/output.txt

Total script time: 1.42 mins

Server is running on http://pr882.butyr.org/templates/basic/
Run the Test Suite: http://pr882.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/b555d9d512f3ac9/output.txt

Total script time: 1.40 mins

**Pull request merged**